### PR TITLE
 Add roxygen comments to add_functions as per #330

### PR DESCRIPTION
**Pull request by RichardPilbery**
Not completely familiar with roxygen comments, so happy to amend with more appropriate tags/text as required.

### CODE REVIEW

File: R/add_r_files.R [hunk 1]
```diff
@@ -83,3 +89,37 @@ add_utils <- function(
...
+#' 
+#' This function as boilerplate roxygen comments 
+#' for fct_ and utils_ files
+#'
+#' @param name The name of the file
+#' @param path The path to the R script where the module will be written. 
+#' @param ext A string denoting the type of file to be created. 
+#' 
+#' @rdname file_creation
+#' @export
```
Comment (ColinFay) [inline R/add_r_files.R:102]: Hey, 

I don't think this should be an exported function — chances are that there would be no reason to do `golem::append_roxygen_comment()` :) 

Can you replace this `@export` by a `@noRd`? 

Thanks :)

**Comment by ColinFay** [comment]
That looks great! Thanks a lot :) 

Can you change the `@export` to `@noRd` as shown in my comment? 

Thanks

**Comment by RichardPilbery** [comment]
No worries. Change made as requested.

File: R/add_r_files.R (hunk 1 above)
Comment (RichardPilbery) [reply R/add_r_files.R]: No worries. New commit on its way.

**Comment by RichardPilbery** [comment]
Apologies, Issue #406 has ended up in this issue as I didn't create a new branch before committing. Can you untangle my mess?

**Comment by ColinFay** [comment]
Dope, thanks !!

**Pull request merged**